client/src/components/Nav.tsx

```diff
@@ -569,6 +576,14 @@ const Navigation = ({
           ))}
         </NavDropdown>
       )}
+      <SidebarLink
+        id="preferences-nav"
+        linkTo="/preferences"
+        handleOnClick={resetPages}
+        setIsMenuLinksOpened={() => setIsMenuLinksOpened(false)}
+      >
+        My Preferences
+      </SidebarLink>
```

Comment:
I think I have a slight preference (pun intended) to put this under the My Work submenu.